src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/EditPersonModal.tsx

```diff
@@ -44,80 +50,254 @@ const ShowExtraText = styled(Typography)(() => ({
+    facebookAccounts: personFacebookAccounts,
+    twitterAccounts: personTwitterAccounts,
+    linkedinAccounts: personLinkedinAccounts,
+    websites: personWebsites,
+    legalFirstName: person.legalFirstName,
+    deceased: person.deceased,
+  };
+
+  const onSubmit = async (attributes: PersonUpdateInput): Promise<void> => {
+    try {
+      await updatePerson({
+        variables: {
+          accountListId,
+          attributes,
+        },
+      });
+      enqueueSnackbar(t('Person updated successfully'), {
+        variant: 'success',
+      });
+      handleClose();
+    } catch (error) {
+      enqueueSnackbar(error.message, { variant: 'error' });
+      throw error;
+    }
+  };
 
   return (
-    <Modal
-      isOpen={isOpen}
-      title={'Edit Person'}
-      handleClose={() => handleOpenModal(false)}
-    >
-      <DialogContent dividers>
-        <ContactEditContainer>
-          <ContactPersonContainer>
-            {/* Name Section */}
-            <PersonName person={person} />
-            {/* Phone Number Section */}
-            <PersonPhoneNumber person={person} />
-            {/* Email Section */}
-            <PersonEmail person={person} />
-            {/* Birthday Section */}
-            <PersonBirthday person={person} />
-            {/* Show More Section */}
-            {!personEditShowMore && (
-              <ShowExtraContainer>
-                <ShowExtraText
-                  variant="subtitle1"
-                  onClick={() => setPersonEditShowMore(true)}
-                >
-                  {t('Show More')}
-                </ShowExtraText>
-              </ShowExtraContainer>
-            )}
-            {/* Start Show More Content */}
-            {personEditShowMore ? <PersonShowMore person={person} /> : null}
-            {/* End Show More Content */}
-
-            {/* Show Less Section */}
-            {personEditShowMore && (
-              <ShowExtraContainer>
-                <ShowExtraText
-                  variant="subtitle1"
-                  onClick={() => setPersonEditShowMore(false)}
-                >
-                  {t('Show Less')}
-                </ShowExtraText>
-              </ShowExtraContainer>
-            )}
-          </ContactPersonContainer>
-        </ContactEditContainer>
-      </DialogContent>
-      <DialogActions>
-        <ContactEditModalFooterButton
-          onClick={() => handleOpenModal(false)}
-          variant="text"
-        >
-          {t('Cancel')}
-        </ContactEditModalFooterButton>
-        <ContactEditModalFooterButton
-          onClick={() => handleOpenModal(false)}
-          variant="text"
-        >
-          {t('Save')}
-        </ContactEditModalFooterButton>
-      </DialogActions>
+    <Modal isOpen={true} title={'Edit Person'} handleClose={handleClose}>
+      <Formik
+        initialValues={initialPerson}
+        validationSchema={personSchema}
+        onSubmit={onSubmit}
+      >
+        {(formikProps): ReactElement => (
```

Comment:
So for the other Formik you split out the props, this one you passed it along. Was that because there's lots of props to pass?

Reply:
Yeah instead of passing possibly five different props for formik, I figured it would be better to just pass the whole formik prop object around and split it out in the component itself.